src/popup/components/TransactionTokenRows.vue

```diff
@@ -8,7 +8,7 @@
       :key="token.symbol"
       :class="[
         'token-row',
-        token.isReceived ? TX_FUNCTIONS.received : TX_FUNCTIONS.sent,
+        token.isReceived ? DIRECTION.received : DIRECTION.sent,
```

Comment:
I think this was a mistake from the beginning to use the constant values here. Those are CSS class names and I think it should be placed here as strings.